### DISCUSSION TITLE
Override the prev/next arrows html output

### DISCUSSION THREAD

**Discussion by chrismagiera** [fancyapps/ui#260] [Q&A] [answered]
Hi,

I couldn’t find any solution in the docs, so I’m asking here:
How do I override the default svg within the prev and next arrows with a custom one?

Is it something in this direction?

```
Fancybox.bind( "[data-fancybox]", {

  template: {
    nextButton: '<svg class="custom-svg"></svg>',
    prevButton: '<svg class="custom-svg"></svg>',
  },

}
```

**Comment by fancyapps** [comment, marked as answer]
Hi,

Object hierarchy is like this: Fancybox -> Carousel ->Navigation plugin. So, it would be like this:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  Carousel : {
    Navigation: {
      prevTpl: '<',
      nextTpl : '>',
    }
  }
});
```

**Reply by chrismagiera** [reply]
Wow – thanks a lot for the quick answer!
